src/connection-manager/wsHandling.ts

```diff
@@ -289,3 +320,70 @@ export async function backtrackAmendmentStatus(): Promise<void> {
     await backtrackNetworkAmendmentStatus(networks, url)
   }
 }
+
+/**
+ * Retrieves and store existing Amendments into amendments_status table.
+ *
+ * @param network - Network name.
+ * @param hostName - Hostname to connect.
+ * @returns Void.
+ */
+async function getAmendmentsFromLedgerEntry(
+  network: string,
+  hostName: string,
+): Promise<void> {
+  const allUrls: string[] = []
```

Comment:
Can we add log at the beginning and ending of this process?

Reply:
Same for other processes as well. Would be something like starting ... and finished ...